services/management/commands/verify_school_districts.py

```diff
@@ -96,5 +73,18 @@ def test__verify_school_units_enclosed(division_units):
                     'start': division.start, 'unit': unit, 'geom': unit.location.wkt,
                     'unit.srid': unit.location.srid, 'div.srid': division.geometry.boundary.srid})
             success = False
-    assert success, (
+    return success, (
         "{} errors \n".format(error_count) + "\n\n".join([pprint.pformat(error, indent=4) for error in error_report]))
+
+
+class Command(BaseCommand):
+    help = """ Verify that imported school district data correctly refers to
```

Comment:
Perhaps instead create a dummy smbackend importer that 1) calls the munigeo division importer first, and 2) then verifies the imported data?

Reply:
Added to backlog. The data verification wasn't run automatically before this PR, so making it run is another task anyway.